### PR TITLE
Starting to add notes from unconference

### PR DESCRIPTION
**Pull request by markyjackson-taulia**
Signed-off-by: markyjackson-taulia <marky.r.jackson@gmail.com>

### CODE REVIEW

File: events/2019/11-contributor-summit/unconference-notes/unconference-kind-ben-the-elder.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+default is kind create cluster --image=/kindtest/default --name cluster_name
+
+writes to default kubeconfig
+you can change this by specifying --kubeconfig=config name
+
+kubectl cluster-info --context kube-kubectl
+
+kind delete cluster --name cluster_name
+
+<<<<<<< HEAD
```
Comment (tpepper) [inline events/2019/11-contributor-summit/unconference-notes/unconference-kind-ben-the-elder.md]: There's a merge conflict here.

Also please squash the three commits.

File: events/2019/11-contributor-summit/unconference-notes/unconference-kind-ben-the-elder.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+with sig-storage they are hoping there is a local storage driver that comes with it. The problem is the deployment mechanism for CSI. PR pending.
+
+on docker with mac containers aren't access via ip.
+
+
+
+Questions:
+
+Has anyone tried to launch kine cluster in pod for testing?
```
Comment (tpepper) [inline events/2019/11-contributor-summit/unconference-notes/unconference-kind-ben-the-elder.md]: Missed this before..."kine cluster" -> "kind cluster"?

Comment (markyjackson-taulia) [reply events/2019/11-contributor-summit/unconference-notes/unconference-kind-ben-the-elder.md]: Fixed @tpepper

**Pull request closed without merging**